### PR TITLE
chore(update): update react-native to 0.47.1

### PR DESCRIPTION
**Pull request by lex111**
All is well, we are waiting for the update of the dependency [react-native-photo-view](https://github.com/alwx/react-native-photo-view) (PR [open](https://github.com/alwx/react-native-photo-view/pull/76)). I hope this will be solved in the near future :blush:

### CODE REVIEW

**Comment by housseindjirdeh** [comment]
Nice looks like that PR was merged in :dancer: 

If you think this should be good to merge, let us know! Thanks for thinking of updating RN :)

File: package.json [hunk 1]
```diff
@@ -47,15 +47,15 @@
     "parse-diff": "^0.4.0",
     "query-string": "^4.3.1",
     "react": "16.0.0-alpha.12",
-    "react-native": "0.46.2",
+    "react-native": "0.47.1",
     "react-native-actionsheet": "^2.2.0",
-    "react-native-code-push": "3.0.1-beta",
+    "react-native-code-push": "v5.0.0-beta",
```
Comment (andrewda) [inline package.json]: Hmm I don't think the `v` is right here.

**Comment by andrewda** [comment]
There's currently a RC available for 0.48.0 – we should update to this for now, but that should be coming soon with quite a few significant changes I believe.

**Comment by lex111** [comment]
@housseindjirdeh the last dependency is updated, ready to merge.

@andrewda we will be updated to 0.48 already after 0.47?

**Comment by andrewda** [comment]
@lex111 Yep - just making sure it's on the radar so we can get it updated as soon as it's released!

**Comment by housseindjirdeh** [comment]
Agreed thank you @andrewda, the sooner we can hop on to updating to 0.48 the better especially if there are possible breaking changes

**Pull request merged**